pkg/cloud/openstack/machine/actuator.go

```diff
@@ -94,51 +99,45 @@ func (oc *OpenstackClient) Create(ctx context.Context, cluster *clusterv1.Cluste
 		return nil
 	}
 
-	// get machine startup script
-	var ok bool
-	userData := []byte{}
-	if providerSpec.UserDataSecret != nil {
-		namespace := providerSpec.UserDataSecret.Namespace
-		if namespace == "" {
-			namespace = machine.Namespace
-		}
-
-		if providerSpec.UserDataSecret.Name == "" {
-			return fmt.Errorf("UserDataSecret name must be provided")
-		}
+	userData, err := getUserDataSecret(kubeClient, providerSpec, machine.Namespace)
+	if err != nil {
+		return oc.handleMachineError(machine, apierrors.CreateMachine(
+			"error creating Openstack instance: %v", err))
+	}
 
-		userDataSecret, err := kubeClient.CoreV1().Secrets(namespace).Get(providerSpec.UserDataSecret.Name, metav1.GetOptions{})
+	isMaster := util.IsControlPlaneMachine(machine)
+	var userDataRendered string
+	if len(userData) > 0 {
+		klog.Info("")
```

Comment:
what's this log for?

Reply:
:joy: ... if I ever find such things *before* creating a PR